modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/_data_set_display.scss

```diff
@@ -1,6 +1,14 @@
 .data-set-display {
 	.data-set-display-management-bar-wrapper {
 		border-bottom: 1px solid $border-color;
```

Comment:
I was expecting this to go away based on [Line below toolbar in channels to be moved to its right place](https://issues.liferay.com/browse/COMMERCE-4675)

Reply:
Yep, that ticket got closed due to this one which includes the fix